lib/identity_job_log_subscriber.rb

```diff
@@ -179,6 +184,10 @@ def trace_id(job)
   def duplicate_cron_error?(ex)
     ex.is_a?(ActiveRecord::RecordNotUnique) && ex.message.include?('(cron_key, cron_at)')
   end
+
+  def should_not_warn?(job, ex)
+    !job.class.try(:warning_messages)&.include(ex.class.name)
```

Comment:
1. There is no method `include` on an Array or Enumerable, it's `#include?`-- did any tests catch that?
2. Doing a simple include check won't look at subclasses of errors, like a `rescue` would, is that something we care about here?

Reply:
Yes, I like this, updates made.